chiselFrontend/src/main/scala/chisel3/core/Data.scala

```diff
@@ -430,3 +435,34 @@ object WireInit {
     x
   }
 }
+
+/** RHS (source) for Invalidate API.
+  * Causes connection logic to emit a DefInvalid when connected to an output port (or wire).
+  */
+object DontCare extends Element(width = UnknownWidth()) {
+  binding = DontCareBinding()
```

Comment:
Consider just calling Data/Element's binding function instead of messing with internal variables. That also prevents rebinding, unless there's a compelling reason to support it

Reply:
Done.